chaos-mesh-action-integrate-chaos-engineering-into-your-ci.md

```diff
@@ -0,0 +1,199 @@
+---
+title: 'chaos-mesh-action: Integrate Chaos Engineering into Your CI'
+author: ['Xiang Wang']
+date: 2020-09-29
+summary: Chaos Mesh is a cloud-native chaos testing platform that orchestrates chaos in Kubernetes environments. This article shares how to use chaos-mesh-action, a GitHub action to integrate Chaos Mesh into the CI process.
+tags: ['Chaos Mesh', 'Chaos Engineering', 'GitHub Action', 'Continuous Integration']
+categories: ['Engineering']
+image: /images/blog/chaos-mesh-action.png
+---
+
+![chaos-mesh-action - Integrate Chaos Engineering into Your CI](/media/chaos-mesh-action.png)
+
+[Chaos Mesh](https://chaos-mesh.org) is a cloud-native chaos testing platform that orchestrates chaos in Kubernetes environments. While it’s well received in the community with its rich fault injection types and easy-to-use dashboard, it was difficult to use Chaos Mesh with end-to-end testing or the continuous integration (CI) process. As a result, problems introduced during system development could not be discovered before the release.
```

Comment:
```suggestion
[Chaos Mesh](https://chaos-mesh.org) is a cloud-native chaos testing platform that orchestrates chaos in Kubernetes environments. While it's well received in the community with its rich fault injection types and easy-to-use dashboard, it was difficult to use Chaos Mesh with end-to-end testing or the continuous integration (CI) process. As a result, problems introduced during system development could not be discovered before the release.
```

Let's use English quotation marks instead of Chinese ones.

Reply:
Fixed. Thx.